core/store/modules/review/actions.ts

```diff
@@ -0,0 +1,92 @@
+import { ActionTree } from "vuex";
+import { quickSearchByQuery } from "core/store/lib/search";
+import { adjustMultistoreApiUrl } from '../../lib/multistore'
+import RootState from "../../types/RootState";
+import ReviewState from "./types/ReviewState";
+import * as types from '../../mutation-types'
+import EventBus from "@vue-storefront/core/plugins/event-bus";
```

Comment:
please use `rootstore` instead of event bus. @DavidRouyer put a lot of effort to get rid of all its occurencies in Vuex stores